pydis_site/templates/events/pages/code-jams/9/_index.html

```diff
@@ -30,12 +30,22 @@ <h3 id="important-dates"><a href="#important-dates">Important Dates</a></h3>
         <li>Sunday, July 31 - Coding portion of the jam ends</li>
         <li>Sunday, August 4 - Code Jam submissions are closed</li>
     </ul>
-    <h3 id="how-to-join"><a href="#how-to-join">How to Join</a></h3>
+
+    <h3 id="qualifier"><a href="#how-to-join">The Qualifier</a></h3>
```

Comment:
Note how `ahref` is still the same, I was hoping this would mean that old links still work correctly.